src/globals/internal/FloatingMenu.js

```diff
@@ -0,0 +1,329 @@
+import warning from 'warning';
```

Comment:
Would like to test this out on other components using floating menu, as part of another task.